src/bulk-import/services/bulk-import.service.ts

```
@@ -0,0 +1,646 @@
+          const tenantRoleMapping: tenantRoleMappingDto = {
+            roleId: '493c04e2-a9db-47f2-b304-503da358d5f4',
+            cohortIds: [], // Set to empty array to satisfy type, but do not include cohortId
+            tenantId: tenantId,
+          };
+          userCreateDto.tenantCohortRoleMapping = [tenantRoleMapping];
+
+          // Keep response for compatibility, but do not log or stringify it
+          const createdUser = await this.createUserForBulk(request, userCreateDto, request.headers.academicyearid);
+
+          // Log only serializable part of createdUser
+          if (createdUser && createdUser.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)